src/lib/web-component/log-view/log-view-element.js

```diff
@@ -3,6 +3,8 @@ import { dateTimeFormat } from '../../formatters.js';
 export class LogView extends HTMLElement {
   constructor() {
     super();
+
+    this.totalLogs = 0;
```

Comment:
This member isn't needed, it can be a local variable.

Reply:
Made it local variable.